pyop2/compilation.py

```diff
@@ -498,13 +499,34 @@ def __init__(self, code, argtypes):
         raise CompilationError("Don't know what compiler to use for platform '%s'" %
                                platform)
     dll = compiler.get_so(code, extension)
+    if isinstance(jitmodule, GlobalKernel):
```

Comment:
Is there ever a case when this is `False`? I'm pretty sure that `jitmodule` is always a `GlobalKernel`.

Reply:
I am pretty sure it is not always a GlobalKernel, see https://github.com/OP2/PyOP2/blob/42812fad033509fdf1a77cf34ed82828f757d75c/pyop2/compilation.py#L471

Reply:
Ah OK.